languages/erlang/exercises/concept/basics/.meta/design.md

```diff
@@ -0,0 +1,29 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to introduce the basics of programming in Erlang.
+
+## Learning Objectives
+
+- Know what a variable is
+- Know that variables are immutable once bound
+- Know how to define a variable
+- Know how to define a function
+- Know how to return a value from a function
+- Know how to call a function
+- Know how to define a module
+- Know how to export a function
+- Know how to use basic mathematical operators
+
+## Out of Scope
+
+- Higher-order functions
+- Anonymous functions
+- Private functions
+
+## Concepts
+
+The concepts this exercise unlocks are:
+
+TBD
```

Comment:
This will likely be:

```suggestion
- `basics`
```

This is how most tracks organize this, as subsequent exercises can then simply have `basics` as a prerequisite.